src/components/roverSelector/RoverSelectorPage.tsx

```diff
@@ -1,9 +1,24 @@
 import React from "react";
+import "./RoverSelectorPage.scss";
+import Opportunity from "/images/OpportunityRover.jpg";
+import Spirit from "/images/SpiritRover.jpg";
+import Curiosity from "/images/CuriosityRover.jpg";
```

Comment:
Ooh, I didn't know you could do this! Could we rename the variables to be lowercase and say "image" please, e.g. `opportunityImage`?